AndroidProjects.yml

```diff
@@ -1,9 +1,9 @@
-name: Beginner-Level-AndroidStudio-Projects
-desc: this repository consists of some basic projects of android studio 
-site: https://github.com/anshulkhattar/Beginner-Level-Android-Studio-Apps
```

Comment:
@anshulkhattar This was fine. Please correct.

Reply:
Done, please check.